reporter-web-app/src/models/WebAppPackage.js

```diff
@@ -86,7 +86,7 @@ class WebAppPackage {
 
     #projectIndexes;
 
-    #purl;
+    #purl = '';
```

Comment:
Why is this needed?

Reply:
Oops, was left-over form debugging removed it now